models/token.go

```diff
@@ -21,11 +21,11 @@ type AccessToken struct {
 	Sha1 string `xorm:"UNIQUE VARCHAR(40)"`
 
 	Created           time.Time `xorm:"-"`
-	CreatedUnix       int64
+	CreatedUnix       int64     `xorm:"INDEX"`
```

Comment:
> a non-clustered index will not help ORDER BY clauses

I thought it would.

@ethantkoenig Since I'm not a databases guru, if you still think we should not have these indexes, feel free to open another PR. I'd merge it. :smiley: